### ISSUE TITLE
[REQUEST] ECAM - Add switching panel / DMC functionality

### ISSUE THREAD

**Issue by krangerich** [flybywiresim/a32nx#1921]
**Is your feature request related to a problem? Please describe.**

Improve realism

**Describe the feature you'd like to see implemented**

* Switching the upper ECAM off (using the brightness switch) should show the E/WD (engine/warning) page on the lower ECAM (instead of the system/status page)
* Turning the ECAM/ND XFR (the rightmost) switch to CAPT or F/O should move the SD (system/status) page to the left or right ND panel, replacing the navigation display. If the upper ECAM is switched off and the lower ECAM shows the E/WD page, the E/WD page is moved instead to the appropriate ND display
* Turning off a PFD (using the PFD brightness switch) should display the PFD on the ND display (on the respective side). 
* Pushing the PFD/ND XFR button (between the PFD / ND brightness knobs) should switch the respective PFD / ND displays

**References**

![image](https://user-images.githubusercontent.com/7890659/99151863-59e80c00-269e-11eb-8901-7a250534e50e.png)
![image](https://user-images.githubusercontent.com/7890659/99151872-6b311880-269e-11eb-9c3c-dc908818c63c.png)
![image](https://user-images.githubusercontent.com/7890659/99151879-784e0780-269e-11eb-814e-2f93fbf8cb95.png)
![image](https://user-images.githubusercontent.com/7890659/99151886-82700600-269e-11eb-9781-b8cb2489b76e.png)

**Additional context**

Discord username (if different from GitHub):
n/a

**Comment by ErikNL80** [comment]
+1 for this request

**Comment by tracernz** [comment]
I did an implementation of this some time back but ultimately parked it as the displays are different sizes in the Asobo 3D model, so some of the displays get clipped. We can do this once our own 3D model is ready.

**Comment by tracernz** [comment]
This is in progress and will be completed when I return from holiday.
